.github/workflows/cross-version-tests.yml

```diff
@@ -77,6 +77,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
+      max-parallel: 10
```

Comment:
Since yesterday, runs for the cross-version tests get stuck at queue for some reason. This change attempts to fix that issue.

Reply:
Doc for `max-parallel`:

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymax-parallel

> The maximum number of jobs that can run simultaneously when using a matrix job strategy. By default, GitHub will maximize the number of jobs run in parallel depending on the available runners on GitHub-hosted virtual machines.